packages/web3-core-requestmanager/src/givenProvider.js

```diff
@@ -34,9 +34,16 @@ try {
 }
 
 // EthereumProvider
+//
+// TODO: removable? This `if` clause looks like an early, subsequently abandoned
```

Comment:
Yes I believe `ethereumProvider` is a pre-EIP1193 artifact. It could make sense to keep it around but not sure if anyone is offering it anymore.